src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -182,20 +182,33 @@ ServerHandlerImp::onHandoff (Session& session,
     {
         if(is_ws)
         {
-            Handoff handoff;
-            auto const ws = session.websocketUpgrade();
+            std::shared_ptr<WSSession> ws;
+            try
+            {
+                ws = session.websocketUpgrade();
+            }
+            catch (std::exception const& e)
+            {
+                JLOG(m_journal.error())
+                    << "Exception upgrading WSS: " << e.what() << "\n";
+                return StatusRequestResponse(request,
+                    boost::beast::http::status::internal_server_error);
```

Comment:
Nitpick: not required to fully scope this guy as you have a `using namespace boost::beast::http`.

Reply:
It doesn't have a `using namespace boost::beast::http` but I'll add one to shorten two instances in this function, thanks.

Reply:
Oops, sorry about that. I saw it on https://github.com/miguelportilla/rippled/blob/03830ad02bf2efdd71cb8c8e2a82a6f0fdecde52/src/ripple/rpc/impl/ServerHandlerImp.cpp#L71 which was right above, not realizing the gap in the diff when viewing on GitHub. Sorry!